api/src/main/java/org/openmrs/module/dhisreport/api/utils/MonthlyPeriod.java

```diff
@@ -38,13 +38,25 @@
     protected Date endDate;
 
     @Override
-    public Date getStart()
+    public void setStartDate( Date startDate )
+    {
+        this.startDate = startDate;
+    }
+
+    @Override
+    public void setEndDate( Date endDate )
+    {
+        this.endDate = endDate;
+    }
+
+    @Override
+    public Date getStartDate()
     {
         return startDate;
     }
 
     @Override
-    public Date getEnd()
+    public Date getEndDate()
```

Comment:
Some of these getter/setter name changes will affect HibernateDHIS2ReportingDAO.java and DHIS2ReportingServiceImpl.java. can you update the methods there as well